dspace/config/modules/requestitem.cfg

```diff
@@ -0,0 +1,43 @@
+#---------------------------------------------------------------#
+#----------------REQUEST ITEM CONFIGURATION---------------------#
+#---------------------------------------------------------------#
+
+# Configuration of request-item. Possible values:
+# all - Anonymous users can request an item
+# logged - Login is mandatory to request an item
+# empty/commented out - request-copy not allowed
+#request.item.type = all
+# Should all Request Copy emails go to the helpdesk instead of the item submitter?
+#request.item.helpdesk.override = false
+# Should a rejection of a copy request send an email back to the requester?
+# Defaults to "true", which means a rejection email is sent back.
+# Setting it to "false" results in a silent rejection.
+#request.item.reject.email = true
+# Bundles to use when granting access and inspecting file size, if "all files" is indicated
+# Default: ORIGINAL
+#request.item.grant.bundles = ORIGINAL
+# By default a granted request will send the file as an attachment.
+# If a secure web link to the bitstream(s) should be sent in some cases, enable the following property
+# Default: false
+#request.item.grant.link = true
+# If request.item.grant.link is enabled, you can specify a minimum file size (in megabytes) to use as
+# a threshold, or else the default attachment functionality will be used instead.
+# If 'all files' was indicated in the request, this threshold will be activated if any of the files
+# meet the configured size minimum.
+# To send links instead of attachments for all files, set this property to 0.
+# Default: 20
+#request.item.grant.link.filesize = 0
+# Valid access periods, in seconds, to allow the approver to select when using links.
+# These are presented to users with friendly labels via i18n keys in the angular frontend.
+# A 0 (or null) access period will be interpreted as 'forever'.
+# The first access period in the list will be the default option in the grant form dropdown.
+# 1 day = 86400
+# 7 days = 604800
+#request.item.grant.link.period = 0
+#request.item.grant.link.period = 120
+#request.item.grant.link.period = 240
+#request.item.grant.link.period = 86400
+#request.item.grant.link.period = 604800
+# Require a captcha for item request creation
+# Default: false
```

Comment:
Should this default to "true" since it's using a semi-"built-in" captcha approach?

Does the captcha also work if you just use the traditional request-a-copy?  Or does it require enabling `request.item.grant.link = true`?

Reply:
It will protect the form regardless of the sharing method, yes. I'll make sure the defaults are all ok, there might be something I can do to further separate it from google recaptcha, too (since i abandoned my "combine captcha" work for lack of time)